index.html

```diff
@@ -15,47 +14,232 @@
   <header>
     <nav>
       <div class="topbar">
-        <a class="ham" href="#"><img src="Assets\ham.svg"></a>
+        <i class="fa fa-bars" aria-hidden="true"></i>
       </div>
-      
     </nav>
   </header>
 
   <main>
 
-  <section>
+  <section id="page1">
     <div class="container">
+      <div class="contentbox">
+        <div class="heading">
+          <h1>Hey there.<br>I'm Victor.<br><span>I’m a software <br> developer</span></h1>
+        </div>
+  
+        <div class="pg1content">
+          <p> I can help you build a product ,<br> feature or website Look through <br>some of my work and experience!<br> If you like what you see and have<br> a project you need coded,<br> 
+            don’t hesitate to contact me.</p>
+        </div>
+  
+        <div class="social">
+          <ul>
+           <li><a href="https://twitter.com/dvcorreia1984"> <img src="./Assets/social/twitter.svg" alt="Twitter"></a></li>
+           <li><a href="https://www.linkedin.com/in/dvcorreia/"><img src="./Assets/social/Linkedin icon.svg" alt="Linkedin"></a></li>
+           <li><a href="#"><img src="./Assets/social/medium.svg" alt="Medium"></a></li>
+           <li><a href="https://github.com/dvcorreia1984"><img src="./Assets/social/github.svg" alt="Github"></a></li>
+           <li><a href="#"><img src="./Assets/social/angellist.svg" alt="Angellist"></a></li>
+          </ul>
+        </div>
 
-      <div class="page1bg"></div>
+        <div class="action">
+          <a href="#page2" class="action-button"><i class="fa fa-chevron-down" aria-hidden="true"></i></a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section id="page2">
+    <div class="contentbox">
+      <header>
+        <h1 class="works">My Recent<br> Works</h1>
```

Comment:
- [ ] As suggested by the previous reviewer,  You have multiple `<h1>` tags on your web page. Kindly change this `<h1>` to other `hx(like h2,h3 etc.)` tag. It is better for your page to have just one `<h1>` tag, this makes your website semantic and easily readable by browsers. 👍